cfep-01.md

```diff
@@ -0,0 +1,135 @@
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+# Abstract
+
+CFEPs are the manner in which the conda-forge community suggests changes to the
+project, and outlines how the suggested changes are reviewed.  This document provides details on what a CFEP is, how to author one, and the workflow used to review these proposals.
+
+# What is a CFEP
+
+CFEP stands for Conda-Forge Enhancement Proposal.  A CFEP is a document which
+outlines a suggested change to how the project operates.  These changes may be
+technical in manner or may deal with social aspects of the project including
+governance and conduct.  The CFEP should provide a concise description of the
+proposed change as well as the rationale for the change.
+
+CFEPs are intended to be the primary mechanism for proposing major changes to
+how conda-forge operates and for documenting both the changes themselves and the
+decision making process.
+
+CFEPs are maintained as text files using Markdown for formatting in the
+[conda-forge/conda-forge-enhancement-proposals](https://github.com/conda-forge/conda-forge-enhancement-proposals)
+repository. Their revision history is a historic record of the proposed change.
+
+# Workflow
+
+The CFEP process begins with an idea for a change in how the conda-forge
+organization operates.  These change can be technical or address the
+social aspects of the organization.  Small changes often do not need a CFEP and
+can be discussed on the
+[conda-forge gitter channel](https://gitter.im/conda-forge/conda-forge.github.io),
+the [conda-forge mailing list](https://groups.google.com/forum/#!forum/conda-forge),
+or as part of a conda-forge meeting.  More involved or controversial changes
+should be submitted as CFEPs.  When it is unclear if a change requires a CFEP,
+ask in one of the above forums and the conda-forge core team should be able to
+provide guidance.  The conda-forge core team may solicit CFEPs on topics of
+concern to the community.  CFEPs should focus on a single issue, broad changes
+should be split into multiple well-focused CFEPs.
+
+Each CFEP must have a champion or champions -- someone who will write the
+CFEP in the format described below, submit the CFEP as a pull request,
+follow the discussion on the pull request, and answer questions.  Before
+beginning to write a proposal, it is best to ascertain if the idea is CFEP-able.
+A discussion on the
+[conda-forge gitter channel](https://gitter.im/conda-forge/conda-forge.github.io),
+the [conda-forge mailing list](https://groups.google.com/forum/#!forum/conda-forge),
+or as part of a conda-forge meeting is the best way to go about this.
+
+Once the champion has asked the conda-forge community as to whether an idea
+has any chance of acceptance, a draft CFEP should be written following the
+template in [CFEP-00](cfep-00.md) and described below.
+
+The CFEP draft may be circulated to interested parties for informal comment and
```

Comment:
This can be dropped. I think we can assume that champions will get feedback on their draft before making the formal submission, so permission isn't needed, and I would like to funnel feedback through CFEP pull requests rather than through informal channels.